README.md

```diff
@@ -354,3 +354,37 @@ The `Table` component requires child components of type `Table.Column`, which pr
 **Usage Example**
 
 [Sample Code](https://github.com/Clever/components/tree/master/docs/TableExample.jsx) ([Live Demo](http://clever.github.io/components/#table))
+
+### Grid
+
+12-column grid component for consistent, simple and flexible layouts.
```

Comment:
added more info to the README - https://github.com/Clever/components/pull/57/commits/7452425f57dcfcce6161be1743de10748623383b